checkout-kata.Core/CheckoutMachine.cs

```diff
@@ -64,19 +65,19 @@ public void Done()
             Logger.Log($"Your total price is: {TotalPrice}");
         }
 
-        /// <summary>
-        /// Calculates all the discounts that are applicable to the items in the shopping basket.
-        /// </summary>
```

Comment:
I see that you have added an Obsolete attribute but this method is no longer used so I would delete it.